docs/src/main/asciidoc/logging.adoc

```diff
@@ -56,24 +51,22 @@ public class ExampleResource {
 ----
 
 NOTE: While JBoss Logging routes log messages into JBoss Log Manager directly, one of your libraries might rely on a different logging API.
-In such cases, you need to use a <<logging-adapters,logging adapter>> to ensure that its log messages are routed to JBoss Log Manager as well.
-
+In such cases, you need to use a <<logging-apis, logging adapters>> to ensure that its log messages are routed to JBoss Log Manager as well.
 
-== Methods of obtaining an application logger
+== Get an application logger
```

Comment:
We are refactoring grammar a lot here. What is the official rule for these section headers (I assume there is one)?

Reply:
Yesh, it's the header convention for reference guides - it must be like: (how to) do something great.

Reply:
https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/doc-reference.adoc#titles-and-headings